jta-and-hibernate-standalone/src/test/java/org/jboss/narayana/quickstart/jta/TestCase.java

```diff
@@ -182,7 +179,7 @@ private void assertEntities(QuickstartEntity... expected) throws Exception {
     }
 
     private List<QuickstartEntity> getEntitiesFromTheDatabase() throws Exception {
-        DataSource dataSource = InitialContext.doLookup("java:/quickstartDataSource");
+        DataSource dataSource = InitialContext.doLookup(TransactionalConnectionProviderTest.DATASOURCE_JNDI);
         Connection connection = dataSource.getConnection(TransactionalConnectionProvider.USERNAME,
```

Comment:
What about replacing TransactionalConnectionProvider with TransactionalConnectionProviderTest in this line @jmfinelli ?

Reply:
Thanks @marcosgopen!